src/exporters/STLExporter.ts

```diff
@@ -131,39 +148,39 @@ STLExporter.prototype = {
       }
     }
 
-    function writeNormal(vA, vB, vC) {
+    function writeNormal(vA: Vector3, vB: Vector3, vC: Vector3): void {
```

Comment:
Same principal as before, I'd like to avoid declaring functions inside methods, if its a function, lets make it a private method imo.

Reply:
That's right, I will fix this as we did `MMDExporter` refactor.